### DISCUSSION TITLE
CSP-J 2021 游记 | ChungZH's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ChungZH/ChungZH.github.io#51] [Comments]
# CSP-J 2021 游记 | ChungZH's Blog

Welcome to my blog!

http://blog.chungzh.cn/articles/csp2021/

**Comment by zirnc** [comment]
test comment

**Comment by aeilot** [comment]
普及组挺简单的，提高组彻底崩了。。。🤦‍♀️去年提高还能 225 的，今年估计100都没有。。。

**Reply by zirnc** [reply]
太强了

**Reply by aeilot** [reply]
然而其实这个225是自测的，那天生病没去考。。。😅洛谷上都说去年提高组是这三年最简单的，今年最难

**Comment by disorderk** [comment]
这就是别人的初二吗_(:з」∠)_，害怕
